load/loader.go

```diff
@@ -120,9 +125,16 @@ func (l *BenchmarkRunner) RunBenchmark(b Benchmark, workQueues uint) {
 // GetBufferedReader returns the buffered Reader that should be used by the loader
 func (l *BenchmarkRunner) GetBufferedReader() *bufio.Reader {
 	if l.br == nil {
-		if len(l.filename) > 0 {
-			l.br = nil // TODO - Support reading from files
+		if len(l.fileName) > 0 {
+			// Read from specified file
+			file, err := os.Open(l.fileName)
+			if err != nil {
+				fatal("cannot open file" + l.fileName)
```

Comment:
this should be the same as the other:
`fatal("cannot open file %s: %v", l.fileName, err)`

Reply:
ok, done